contracts/util/ZeroCollateralCommon.sol

```diff
@@ -78,6 +78,7 @@ library ZeroCollateralCommon {
         uint256 lastCollateralIn;
         uint256 principalOwed;
         uint256 interestOwed;
+        uint256 borrowedAmount;
```

Comment:
This is the same as `principalOwed`?

Reply:
Yes, when the user takes out a loan, the borrowed amount, and principalOwed are set. But after the first repayment, we don't know the original borrowed amount (the principalOwed is modified).
So, then, we could aggregate data based on all the borrowed amounts.